packages/packages.go

```diff
@@ -44,3 +48,11 @@ func convertGopkgNameToPurl(name string) (rename string) {
 	}
 	return
 }
+
+func GimmeAPurl(name string, version string) string {
```

Comment:
I can change this name, but I was laughing as I typed it

Reply:
```suggestion
func gimmeAPURL(name string, version string) string {

Reply:
Haha YESSSSS